srcpkgs/yed/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'yed'
+pkgname=yed
+version=3.19.1.1
+revision=1
+archs=noarch
+wrksrc="$pkgname-${version}"
+hostmakedepends="bsdtar unzip"
+depends="virtual?java-environment"
+short_desc="Very powerful graph editor written in Java"
+maintainer="Joseph Benden <joe@benden.us>"
+license="custom:Commercial"
+homepage="https://www.yworks.com/products/yed/"
+distfiles="https://www.yworks.com/resources/yed/demo/yEd-${version}.zip"
+checksum="3639da2650976bffcd43a8ea830727bae86478e95ceb4d68ad6363a42a8f1efb"
+
+repository=nonfree
```

Comment:
if we aren't allowed to distribute it, set `restricted=yes`

Reply:
Thank you... Apparently, still learning Void Linux ropes... :)